.github/workflows/pull_request.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright 2024 NWChemEx Community
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: .github Pull Request Workflow
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  Common-Pull-Request:
+    uses: NWChemEx/.github/.github/workflows/common_pull_request.yaml@master
+    with:
+      config_file: '.github/.licenserc.yaml'
+      source_dir: ''
+      compilers: '["gcc-11", "clang-14"]'
+      doc_target: 'structure_finder_cxx_api'
```

Comment:
`test_build_docs` isn't going to pass with this value for `doc_target` until some changes to the `CMakeLists.txt` are made (see comment on that file) and some minimal Sphinx documentation is added. `doc_target` can be set to `''` to skip the test for now, if you want to add the documentation later.